src/docs/get-started/install/_android-setup.md

```diff
@@ -52,7 +52,7 @@ follow these steps:
  1. Choose a device definition and select **Next**.
  1. Select one or more system images for the Android versions you want
     to emulate, and select **Next**.
-    An _x86_ or _x86\_64_ image is recommended.
+    A _x86_ or _x86\_64_ image is recommended.
```

Comment:
This is nit-picky, I know, but I think it would make more sense to keep this as "An _x86...". Just because of how it's actually pronounced, saying "A ex-eighty-six" doesn't make sense while "An ex-eighty-six" does.

Reply:
@micpap25 is quite right. It should be "An", not "A".

@ianloic, could you remove the spellcheck-cli from this PR? I would very much like to merge the fixed errors.  :)

Reply:
Will sort this within the week of that's ok. Just moved job and waiting for new laptop to be posted out 😃

Reply:
That sounds good, @jimmycasey. And congrats on the new job!